src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -73,9 +73,12 @@
     "res/layout/main.xml": {
       "Size": 544
     },
+    "res/xml/splits0.xml": {
+      "Size": 120
+    },
     "resources.arsc": {
-      "Size": 1724
+      "Size": 1904
     }
   },
-  "PackageSize": 2709331
+  "PackageSize": 2705300
```

Comment:
How is the size of all the files bigger, but the package smaller?

Reply:
One of the changes recently was to default `Release` builds to create both an `.aab` and an `.apk`.

We did this by making `bundletool` output a "universal" `.apk` *from* the App Bundle. So I think we're just seeing differences from this change, that didn't show up in the original PR.